### PR TITLE
Rewrite the installation guide for clarity and flow.

### PR DESCRIPTION
**Pull request by spomorski**
Rewrites the Tekton Pipelines installation guide for better clarity and flow.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -1,159 +1,162 @@
...
 
-Use this page to add the component to an existing Kubernetes cluster.
+This guide explains how to install Tekton Pipelines. It covers the following topics:
 
-* [Pre-requisites](#pre-requisites)
-* [Versions](#versions)
-* [Installing Tekton Pipelines](#installing-tekton-pipelines)
+* [Before you begin](#before-you-begin)
+* [Installing Tekton Pipelines on Kubernetes](#installing-tekton-pipelines-on-kubernetes)
 * [Installing Tekton PIpelines on OpenShift/MiniShift](#installing-tekton-pipelines-on-openshiftminishift)
```
Comment (sbwsg) [inline docs/install.md]: Nit of thing you didn't change :angel: : `PIpelines` -> `Pipelines` (lowercase i)

File: docs/install.md [hunk 2]
```diff
@@ -1,159 +1,162 @@
...
+  service account JSON file.
+- `bucket.service.account.field.name` - the name of the environment variable to use when specifying the
   secret path. Defaults to `GOOGLE_APPLICATION_CREDENTIALS`. Set to `BOTO_CONFIG` if using S3 instead of GCS.
+  
+**Important:** Configure your bucket's retention policy to delete all files after your `Tasks` finish running.
 
-*Note:* When using an S3 bucket, there is a restriction that the bucket is located in the us-east-1 region.
-This is a limitation coming from using [gsutil](https://cloud.google.com/storage/docs/gsutil) with a boto configuration
-behind the scene to access the S3 bucket.
+**Note:** You can only use an S3 bucket locakted in the `us-east-1` region. This is a limitation of [`gsutil`](https://cloud.google.com/storage/docs/gsutil) running a `boto` configuration behind the scenes to access the S3 bucket.
```
Comment (sbwsg) [inline docs/install.md]: `locakted` -> `located`

File: docs/install.md [hunk 3]
```diff
@@ -182,25 +185,16 @@ data:
...
-to a bucket, or if the the cluster is running in multiple zones, the access to
-the persistent volume can fail.
-
-### Overriding default ServiceAccount, Timeout or PodTemplate used for TaskRun and PipelineRun
+## Customizing basic execution parameters
 
-The ConfigMap `config-defaults` can be used to override default values.
-You can override the default service account, the default timeout, and the
-default pod template applied to `TaskRun` and `PipelineRun`.
+You can specify your own values that replace the default service account (`ServiceAccount`), timeout (`Timeout`), and Pod template (`PodTemplate`) values used by Tekton Pipelines in `TaskRun` and `PipelineRun` definitions. To do so, use the ConfigMap `config-defaults`.
```
Comment (sbwsg) [inline docs/install.md]: Suggest "create the ConfigMap" at the end here rather than "use the ConfigMap".  Or maybe "modify"?  "Use" feels like the wrong verb to me hre since the operator will need to take a mutating action on the ConfigMap.

File: docs/install.md (hunk 1 above)
Comment (spomorski) [reply docs/install.md]: Hey, that's how I make sure reviewers actually read the doc... just kidding! Thank you, I often miss small things like this after a whole day of writing. Fixed. :)

File: docs/install.md (hunk 2 above)
Comment (spomorski) [reply docs/install.md]: Derp! Fixed.

File: docs/install.md (hunk 3 above)
Comment (spomorski) [reply docs/install.md]: Thank you! I don't know exactly what a ConfigMap is so I didn't know the correct verb to pair it with. Changed to modify and added a bit of context.

File: docs/install.md [hunk 4]
```diff
@@ -1,159 +1,162 @@
...
+
+   * **[Nightly](../tekton/README.md#nightly-releases)** - may contain bugs,
+     install at your own risk. Nightlies live at `gcr.io/tekton-nightly`.
+   * **[`HEAD`]** - this is the bleeding edge. It contains unreleased code that may result
+     in unpredictable behavior. To get started, see the [development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md) instead of this page.
+
+2. If you don't have an existing Kubernetes cluster, set one up, version 1.15 or later:
 
    ```bash
-   # Example cluster creation command on GKE
```
Comment (afrittoli) [inline docs/install.md]: I would prefer not to lose this comment. The command in GKE specific and it's not the specifically recommended way to install Tekton. Tekton will work on any on any k8s cluster (with a recent enough version).

File: docs/install.md [hunk 5]
```diff
@@ -1,159 +1,162 @@
...
 [Security Context Constraints](https://docs.openshift.com/container-platform/3.11/admin_guide/manage_scc.html)
-for more information_
+for more information.
 
-1. First, login as a user with `cluster-admin` privileges. The following example
+1. Log on as a user with `cluster-admin` privileges. The following example
    uses the default `system:admin` user (`admin:admin` for MiniShift):
 
    ```bash
-   # For MiniShift: oc login -u admin:admin
```
Comment (afrittoli) [inline docs/install.md]: Why is this removed?

**Comment by spomorski** [comment]
> Nice thank you!
> You may need a rebase though.
> See also #2123 - it would be good to integrate with it - either of the two will need a rebase

Since my rewrite is a lot more extensive, it's better to integrate your addition into my PR.

File: docs/install.md (hunk 4 above)
Comment (spomorski) [reply docs/install.md]: Added back this comment.

File: docs/install.md (hunk 5 above)
Comment (spomorski) [reply docs/install.md]: Added back this comment.

File: docs/install.md [hunk 6]
```diff
@@ -1,167 +1,172 @@
...
-## Pre-requisites
+## Before you begin
 
-1. A Kubernetes cluster version 1.15 or later (_if you don't have an existing
-   cluster_):
+1. Choose the version of Tekton Pipelines you want to install. You have the following options:
+
+   * **[Official](https://github.com/tektoncd/pipeline/releases)** - install this unless you have
+     a specific reason to go for a different release.
+
```
Comment (bobcatfish) [inline docs/install.md]: extra newline?

I AM SO SORRY

**Comment by bobcatfish** [comment]
/hold

for commit squash :D
